lib/lltz_ir/expr.ml

```diff
@@ -4,10 +4,10 @@ open Grace
 type micheline = (Range.t, string) Tezos_micheline.Micheline.node
 
 module T = struct
-  type var = Var of string
+  type var = Var of string 
```

Comment:
I think you added an unwanted trailing white space. It's a problem for diffs and people like me whose editor removes those trailing spaces upon saving. There the same thing at line 10.